code/src/components/Header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { tasks } from '../reducers/tasks'
+import { useSelector } from 'react-redux'
+import './header.css'
+
+export const Header = () => {
+
+const quantity = useSelector((state) => state.tasks.items )
+const totalQuantity = quantity.length
+const moment = require('moment');
+
+
+return (
+   <header>
+
+    {totalQuantity === 0 && (
+        <div className="today">
+            <h1>Todo</h1>
+            <p>{moment().format('MMMM Do, YYYY')}</p>
+        </div>
```

Comment:
since this part is shown in both cases, just write it once. Remember to keep the code DRY.